bot/exts/moderation/modpings.py

```diff
@@ -51,10 +52,15 @@ def __init__(self, bot: Bot):
             event_loop=self.bot.loop,
         )
 
-    async def reschedule_roles(self) -> None:
-        """Reschedule moderators role re-apply times."""
+    async def fetch_guild(self) -> None:
+        """Fetch the guild."""
         await self.bot.wait_until_guild_available()
         self.guild = self.bot.get_guild(Guild.id)
+
+    async def reschedule_roles(self) -> None:
+        """Reschedule moderators role re-apply times."""
+        if not self.guild:
+            await self.fetch_guild()
```

Comment:
Just make this `await self.fetch_guild_task`

Reply:
TIL

Reply:
1dbedb2e